### PR TITLE
Updating paths for Ubiquiti/AirMAX MIB downloads

### PR DESCRIPTION
**Pull request by RustyBower**
@brian-brazil

### CODE REVIEW

**Comment by SuperQ** [comment]
@RustyBower Glad they're finally fixing their MIBs.  Do you know if UBNT has a canonical list of MIB URLs somewhere?

File: generator/README.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ Put the extracted mibs in a location NetSNMP can read them from. `$HOME/.snmp/mi
 * UCD-SNMP-MIB (Net-SNMP): http://www.net-snmp.org/docs/mibs/UCD-SNMP-MIB.txt
 * Ubiquiti Networks: http://dl.ubnt-ut.com/snmp/UBNT-MIB
                      http://dl.ubnt-ut.com/snmp/UBNT-UniFi-MIB
-                     https://community.ubnt.com/ubnt/attachments/ubnt/airMAX_FAQ/84/3/ubnt-mib.04.06.2017.zip
+                     https://www.ubnt.com/downloads/firmwares/airos-ubnt-mib/ubnt-mib.zip
```
Comment (SuperQ) [inline generator/README.md]: I just looked at my test more closely, it looks like this URL is 301'd to `https://dl.ubnt.com/firmwares/airos-ubnt-mib/ubnt-mib.zip`.  I would suggest using this URL.

Comment (RustyBower) [reply generator/README.md]: LGTM

**Comment by RustyBower** [comment]
Unfortunately the only place I found a link to the .zip of MIBs was this random community thread (https://community.ubnt.com/t5/airMAX-Frequently-Asked/airMAX-Where-can-I-find-the-latest-MIB-for-airOS/ta-p/1294379).

I assume (perhaps wrongly) that the latest will be there, since we would no longer be specifying a generated date in the zip name

File: generator/README.md (hunk 1 above)
Comment (RustyBower) [reply generator/README.md]: Updated to pull from that final redirect

**Pull request merged**